### PR TITLE
CharNumber function

### PR DESCRIPTION
**Pull request by Hargeon**
- [x] Implement a function that returns a number of characters in the string
- [x] Cover code with unit tests
- [x] Add examples for function to the test file(added in README)
- [x] Create a new module release

### CODE REVIEW

File: strings_helper.go [hunk 1]
```diff
@@ -18,3 +19,14 @@ func Reverse(str string) string {
 
 	return string(reversedStr)
 }
+
+// CharNumber returns a number of characters in the string
+func CharNumber(str string) int {
+	var charLen int
+	for _, char := range str {
+		if unicode.IsLetter(char) {
```
Comment (gordonBusyman) [inline strings_helper.go:27]: What is this `IF` for?

File: strings_helper_test.go [hunk 2]
```diff
@@ -65,3 +65,75 @@ func testReverseFunc(inputStr, expected string) func(*testing.T) {
...
+		},
+		{
+			name:   "None ASCII phrase with different symbols",
+			input:  "Очень __*/ 459 больш78ая строка   6266",
+			result: 18,
+		},
+		{
+			name:   "Chinese characters with symbols",
+			input:  "帶884數字___ ***的漢字  369",
+			result: 6,
```
Comment (gordonBusyman) [inline strings_helper_test.go:118]: 20

File: strings_helper_test.go [hunk 3]
```diff
@@ -65,3 +65,75 @@ func testReverseFunc(inputStr, expected string) func(*testing.T) {
...
+		},
+		{
+			name:   "None ASCII phrase with emoji",
+			input:  "Это вулкан - 🌋, а не брокколи 🥦",
+			result: 19,
```
Comment (gordonBusyman) [inline strings_helper_test.go:123]: 31

File: strings_helper_test.go [hunk 4]
```diff
@@ -65,3 +65,75 @@ func testReverseFunc(inputStr, expected string) func(*testing.T) {
...
+		},
+	}
+
+	for _, testCase := range cases {
+		t.Run(testCase.name, testCharNumber(testCase.input, testCase.result))
+	}
+}
```
Comment (gordonBusyman) [inline strings_helper_test.go:130]: The `testCharNumber()` can be anonymous function in t.Run()

File: strings_helper_test.go [hunk 5]
```diff
@@ -65,3 +65,75 @@ func testReverseFunc(inputStr, expected string) func(*testing.T) {
...
+
+func testCharNumber(inputStr string, expectedLength int) func(*testing.T) {
```
Comment (gordonBusyman) [inline strings_helper_test.go:132]: No need to have this function

File: strings_helper.go [hunk 6]
```diff
@@ -18,3 +19,14 @@ func Reverse(str string) string {
 
 	return string(reversedStr)
 }
+
+// CharNumber returns a number of characters in the string
+func CharNumber(str string) int {
```
Comment (gordonBusyman) [inline strings_helper.go:24]: What does this function return?

**Review by gordonBusyman — CHANGES_REQUESTED** [review]
Also please add a description with 
- [ ] tasks

**Review by papahmsolo — CHANGES_REQUESTED** [review]
The code looks good!

My two cents about naming: take a look at the `Package Naming` section in [this article](https://rakyll.org/style-packages/) you can also browse other packages in the standard library for the examples of the package names. Words like `helper/common` does not add much meaning to the name.

File: strings_helper.go (hunk 6 above)
Comment (Hargeon) [reply strings_helper.go:23]: CharNumber returned the number of letters. Fixed it

File: strings_helper_test.go (hunk 5 above)
Comment (Hargeon) [reply strings_helper_test.go]: Done

File: strings_helper_test.go (hunk 4 above)
Comment (Hargeon) [reply strings_helper_test.go]: Done

File: strings_helper_test.go (hunk 3 above)
Comment (Hargeon) [reply strings_helper_test.go]: Done

File: strings_helper_test.go (hunk 2 above)
Comment (Hargeon) [reply strings_helper_test.go]: Done. But there are 21 characters in the phrase

File: strings_helper.go (hunk 1 above)
Comment (Hargeon) [reply strings_helper.go]: Filtered letters. Fixed it

File: strings_helper.go (hunk 6 above)
Comment (gordonBusyman) [reply strings_helper.go:23]: The comment was mostly to rethink the func name.. I was thinking about smth like `Length`, but this is okey as well

File: README.md [hunk 7]
```diff
@@ -7,6 +7,11 @@ A Go Package That Makes Strings Easier.
 go get -u github.com/Hargeon/stringhlp
 ```
 
+## Tasks
+- [x] Implement a function that reverses a string.
+- [x] Implement a function that returns a number of characters in the string.
+- [x] Cover code with unit tests.
+
```
Comment (gordonBusyman) [inline README.md:14]: Remove this from `README`

**Review by gordonBusyman — APPROVED** [review]
Just one comment

File: strings_helper.go (hunk 6 above)
Comment (Hargeon) [reply strings_helper.go:23]: Ok

**Pull request merged**